webapp/publisher/snaps/views.py

```diff
@@ -451,11 +455,11 @@ def post_register_name_json():
 @publisher_snaps.route("/register-name-dispute")
 @login_required
 def get_register_name_dispute():
-    user = publisher_api.get_account(flask.session)
+    stores = admin_api.get_stores(flask.session)
```

Comment:
The account no longer has the list of stores, so get the list of stores directly from the API